README.md

```diff
@@ -1,75 +1,71 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Capstone Project
 
-# Project Name
+> This project will be a replica of CC-Global-Summit-2015 website from https://www.behance.net/ gallery. It's a capstone project of the skills and knowledge I have obtained during my first module at Microverse.
 
-> Description the project.
+> It consists of two pages (home and about pages) and these are responsive depending on the screen side.
+
+> It is accessible to all kinds of readers because it's designed with the highly recommended semantic tags, amazing color contrasts, photos and icons.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages - HTML, CSS and JavaScript
+- Frameworks - None
+- Technologies used - Visual Studio Code, Git and Github, Chrome (Browser and it's Dev Tools), Accessibility Tools (Wave Evaluation ool and Web Developer)
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://inspecta.github.io/Capstone-web-project/)
```

Comment:
- [ ] The live demo here is broken so kindly make sure that it is working fine then submit a new pull request because it is one of the requirements 🔥🔥💪🏻💪🏻 


![image](https://user-images.githubusercontent.com/58553711/190532909-62cf877b-e920-4925-b8e0-92f071f5e22a.png)

Reply:
Thank you @Reem-lab. I'll fix the live demo link right away.